plugins/depthCamera/src/DepthCamera.cc

```diff
@@ -137,6 +137,7 @@ void GazeboYarpDepthCamera::Load(sensors::SensorPtr _sensor, sdf::ElementPtr _sd
             yError()<<"GazeboYarpDepthCamera: failed setting scopedDeviceName(=" << scopedDeviceName << ")";
             return;
         }
+        yDebug() << "GazeboYarpDepthCamera: register device:" << scopedDeviceName;
```

Comment:
Is this strictly necessary? The device output of the gazebo yarp plugins is already quite verbose, if this was used for debugging I think we can remove it now.

Reply:
I think it is quite important to be able to search in the log for the name of the registered device (which is why I enabled it for the depthCamera and the doubleLaser). When creating a yarprobotinterface xml file, you have no other clue, unless this line is printed...

Reply:
If this is an important information to print, can we put this as `yInfo` instead?

Reply:
Furthermore, it may not be clear to user what "device" refers to, perhaps we could be a bit more explicit and:
```suggestion
        yDebug() << "GazeboYarpDepthCamera: register YARP device with instance name:" << scopedDeviceName;
```